README.md

```diff
@@ -7,7 +7,7 @@
 [![Gem Version](https://badge.fury.io/rb/defra_ruby_area.svg)](https://badge.fury.io/rb/defra_ruby_area)
 [![Licence](https://img.shields.io/badge/Licence-OGLv3-blue.svg)](http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3)
 
-This ruby gem provides a means of looking up an Environment Agency Administrative boundary area from a GIS Web Feature Service (WFS). Provided with a valid [easting and northing](https://en.wikipedia.org/wiki/Easting_and_northing) it will query the WFS and return the various properties for the area if a match is found.
+This ruby gem provides a means of looking up an Environment Agency Administrative boundary area from a GIS Web Feature Service (WFS). Provided with a valid [easting and northing](https://en.wikipedia.org/wiki/Easting_and_northing) it will query the WFS and return various properties for the matching area found.
```

Comment:
```suggestion
This ruby gem provides a means of looking up an Environment Agency Administrative boundary area from a GIS Web Feature Service (WFS). Provided with a valid [easting and northing](https://en.wikipedia.org/wiki/Easting_and_northing) it will query the WFS and return various properties for the matching areas found.
```

Reply:
Yeah, I thought about that. I came to the conclusion that in probably 99% of cases there will be only one result, so as the opening description I would stick to talking about the gem in that context. I then cover multiple results as a separate thing later.

However I'm happy to change if its clearer and more consistent with everything else. I'd just change a bit more of the opening description if we go this route though

```markdown
This ruby gem provides a means of looking up Environment Agency Administrative boundary areas from a GIS Web Feature Service (WFS). Provided with a valid [easting and northing](https://en.wikipedia.org/wiki/Easting_and_northing) it will query the WFS and return various properties for each matching area found.
```

Reply:
I don't really have a strong opinion about this, but I like the description you are suggesting at the bottom of your comment much more for consistency :)